### DISCUSSION TITLE
REALITY的utls指纹可以选择random吗

### DISCUSSION THREAD

**Discussion by askdnl** [XTLS/Xray-core#2877] [Q&A] [answered]
我现在使用的是vless，vision，reality，我能在服务端和客户端都选择random吗？谢谢

**Comment by chise0713** [comment, marked as answer]
服务端你设置指纹没用，服务端是看REALITY的dest的；客户端你随意，但是未知的指纹可能会带来意想不到的后果

**Reply by askdnl** [reply]
ok，谢谢大佬解答

**Reply by chise0713** [reply]
> ~~考虑把我mark为answer吗，业绩~~

**Comment by chika0801** [comment]
建议你用chrome总没有错。不推荐用random

**Reply by askdnl** [reply]
谢谢大佬指教
